Documentation.docc/Documentation.md

```diff
@@ -0,0 +1,81 @@
+# ``RevenueCat``
+
+😻 In-App Subscriptions Made Easy 😻
+
+## Overview
+
+RevenueCat is a powerful, reliable, and free to use in-app purchase server with cross-platform support. Our open-source framework provides a backend and a wrapper around StoreKit and Google Play Billing to make implementing in-app purchases and subscriptions easy. 
+
+Whether you are building a new app or already have millions of customers, you can use RevenueCat to:
+
+  * Fetch products, make purchases, and check subscription status with our [native SDKs](https://docs.revenuecat.com/docs/installation). 
+  * Host and [configure products](https://docs.revenuecat.com/docs/entitlements) remotely from our dashboard. 
+  * Analyze the most important metrics for your app business [in one place](https://docs.revenuecat.com/docs/charts).
+  * See customer transaction histories, chart lifetime value, and [grant promotional subscriptions](https://docs.revenuecat.com/docs/customers).
+  * Get notified of real-time events through [webhooks](https://docs.revenuecat.com/docs/webhooks).
+  * Send enriched purchase events to analytics and attribution tools with our easy integrations.
+
+Sign up to [get started for free](https://app.revenuecat.com/signup).
+
+### RevenueCat SDK Features
+|   | RevenueCat |
+| --- | --- |
+✅ | Server-side receipt validation
+➡️ | [Webhooks](https://docs.revenuecat.com/docs/webhooks) - enhanced server-to-server communication with events for purchases, renewals, cancellations, and more
+🖥 | macOS support
+🎯 | Subscription status tracking - know whether a user is subscribed whether they're on iOS, Android or web
+📊 | Analytics - automatic calculation of metrics like conversion, mrr, and churn
+📝 | [Online documentation](https://docs.revenuecat.com/docs) up to date
+🔀 | [Integrations](https://www.revenuecat.com/integrations) - over a dozen integrations to easily send purchase data where you need it
+💯 | Well maintained - [frequent releases](https://github.com/RevenueCat/purchases-ios/releases)
+📮 | Great support - [Help Center](https://community.revenuecat.com)
+
+## Getting Started
+For more detailed information, you can view our complete documentation at [docs.revenuecat.com](https://docs.revenuecat.com/docs).
+
+Or browse our iOS sample apps:
+- [MagicWeather](Examples/MagicWeather)
+- [MagicWeather SwiftUI](Examples/MagicWeatherSwiftUI)
+
+## Topics
```

Comment:
These topics organize how the docs are shown, and it's pretty neat:

<img width="1172" alt="image" src="https://user-images.githubusercontent.com/3922667/150359220-0a66107f-475c-4198-8350-a8186dbe6bf6.png">
<img width="503" alt="image" src="https://user-images.githubusercontent.com/3922667/150359242-b8ceb7ce-93ee-42d0-aa58-edd907dfb98c.png">